sympy/functions/special/bessel.py

```diff
@@ -593,6 +593,14 @@ def fdiff(self, argindex=2):
             self * (self.order + 1)/self.argument
 
 
+def _jn(n, z):
+    return fn(n, z)*sin(z) + (-1)**(n + 1)*fn(-n - 1, z)*cos(z)
+
+
+def _yn(n, z):
+    return (-1)**(n + 1) * _jn(-n - 1, z)
```

Comment:
Perhaps this could be written as a combination ``sin`` and ``cos`` in the same way as ``_jn`` above. It would look more symmetric and self-documenting in addition to saving some calls.